docs/setup/overview.md

```diff
@@ -32,7 +30,7 @@ There are three variants of the DSK Agent available:
 
 ### DSK Agent Edge {#dsk-agent-edge}
 
-The DSK Agent Edge is the most common Agent variant and is perfect for all software-based use-cases. Learn how to set it up in our [**specific setup guide**](/docs/setup/agent/overview.md).
+The DSK Agent Edge is our containerized, software-based solution perfect for all devices/environments offering a [container runtime](./agent/overview.md#software-requirements-software-requirements). Learn more in our [**setup guide**](./agent/overview.md).
```

Comment:
Links should be bold

Reply:
try to adapt at all touched files.

would be great if we move such general styling into the theme of docusaurus at some point.